src/clj/puppetlabs/services/jruby/jruby_puppet_agents.clj

```diff
@@ -161,7 +161,8 @@
   (let [{:keys [config pool-state]} pool-context
         new-pool-state (jruby-internal/create-pool-from-config config)
         old-pool @pool-state]
-    (swap-and-drain-pool! pool-context old-pool new-pool-state true)))
+    (swap-and-drain-pool! pool-context old-pool new-pool-state true)
+    (println "*** done flushing and repopulating pool")))
```

Comment:
Definitely not.  :smile:   I'm going to prune off this last "println" commit from the PR before we consider merging this.  I just put this in temporarily to see if I could get the tests to fail in Travis with some more insight around where the fail is occurring.